web/src/components/wizard/InstallWizard.tsx

```diff
@@ -14,29 +13,19 @@ const InstallWizard: React.FC = () => {
   const { text } = useWizardMode();
 
   const goToNextStep = () => {
-    const steps: WizardStep[] = ["welcome", "setup", "validation", "installation", "completion"];
+    const steps: WizardStep[] = ["welcome", "setup", "installation", "completion"];
     const currentIndex = steps.indexOf(currentStep);
     if (currentIndex < steps.length - 1) {
       setCurrentStep(steps[currentIndex + 1]);
     }
   };
 
-  const goToPreviousStep = () => {
-    const steps: WizardStep[] = ["welcome", "setup", "validation", "installation", "completion"];
-    const currentIndex = steps.indexOf(currentStep);
-    if (currentIndex > 0) {
-      setCurrentStep(steps[currentIndex - 1]);
-    }
-  };
-
```

Comment:
I removed this function onBack functionality from the SetupStep because it looks like you can only reach the setup page if you're already authenticated, and the WelcomeStep automatically redirects authenticated users back to SetupStep, so the back button effectively didn't do anything.